backend/btrixcloud/crawlmanager.py

```diff
@@ -118,18 +118,54 @@ async def run_delete_org_job(
         backend_image: str,
         pull_policy: str,
         existing_job_id: Optional[str] = None,
-    ):
+    ) -> str:
         """run job to delete org and all of its data"""
 
         if existing_job_id:
             job_id = existing_job_id
         else:
             job_id = f"delete-org-{oid}-{secrets.token_hex(5)}"
 
+        return await self._run_bg_job_with_ops_classes(
+            oid, backend_image, pull_policy, job_id, job_type=BgJobType.DELETE_ORG.value
+        )
+
+    async def run_recalculate_org_stats_job(
```

Comment:
I wonder if we should perhaps make a more generic run_bg_job here  if most of the params are the same - but can be added in a later refactor as we add more jobs, so shouldn't hold off merge.

Reply:
Oh I see, I guess there already is a more generic method, maybe not needed at all, we'll see!

Reply:
Yes there's a generic private method already, the public methods here really just set the job id to what we want it to be.